src/modules/EdgeDetect/EdgeUtils.js

```diff
@@ -147,17 +182,24 @@ function nonMaxSupress(pixels, grads, angles) {
     }
   }
 }
-// Converts radians to degrees
+
+
+/**
+ * @method convertToDegrees
+ * @description converts the given angle(in radians) to degrees.
+ * @param {Number} radians Angle in radians.
+ * @returns {Number} Angle in degrees.
+ */
 var convertToDegrees = radians => (radians * 180) / Math.PI;
 
-// Finds the max value in a 2d array like grads
+// Finds the max value in a 2d array like grads.
 var findMaxInMatrix = arr => Math.max(...arr.map(el => el.map(val => val ? val : 0)).map(el => Math.max(...el)));
 
-// Applies the double threshold to the image
+// Applies the double threshold to the image.
 function doubleThreshold(pixels, highThresholdRatio, lowThresholdRatio, grads, strongEdgePixels, weakEdgePixels) {
 
-  const highThreshold = findMaxInMatrix(grads) * highThresholdRatio,
-    lowThreshold = highThreshold * lowThresholdRatio;
+  const highThreshold = findMaxInMatrix(grads) * highThresholdRatio, // High Threshold relative to the strongest edge.
```

Comment:
I think you forget to remove full stop from phrases in this pr

Reply:
Yes, I had forgotten to do that but I have pushed a fix now.